src/packet/rustq/src/lib.rs

```diff
@@ -175,27 +184,31 @@ impl WrapperPacketQueue {
         (match &self.inner {
             WrapperPacketQueueInner::ClassTokenBucket(q) => q.0.len_bytes(),
             WrapperPacketQueueInner::DeficitRoundRobin(q) => q.0.len_bytes(),
+            WrapperPacketQueueInner::TrafficPolicer(q) => q.len_bytes(),
         }) - overhead
     }
 
     pub fn qsize_packets(&self) -> usize {
         match &self.inner {
             WrapperPacketQueueInner::ClassTokenBucket(q) => q.0.len_packets(),
             WrapperPacketQueueInner::DeficitRoundRobin(q) => q.0.len_packets(),
+            WrapperPacketQueueInner::TrafficPolicer(q) => q.len_packets(),
         }
     }
 
     pub fn to_string(&self, out: Pin<&mut CxxString>) {
         let s = match &self.inner {
             WrapperPacketQueueInner::ClassTokenBucket(q) => format!("{:?}", q),
             WrapperPacketQueueInner::DeficitRoundRobin(q) => format!("{:?}", q),
+            WrapperPacketQueueInner::TrafficPolicer(q) => format!("{:?}", q),
         };
 
         out.push_str(&s);
     }
 }
 
 use hwfq::{scheduler::htb::ClassedTokenBucket, scheduler::Drr, Pkt, Scheduler};
+use tp::TrafficPolicer;
```

Comment:
Probably would be confusing to have this here - the import is actually used above on line 55. The reason the `use hwfq::*` imports are at this point in the file is that the above parts don't use those types.
So, we should move `use tp::TrafficPolicer` to the top of the file.